### ISSUE TITLE
Make https://eessi.github.io/ (webserver root) work

### ISSUE THREAD

**Issue by robberteggermont** [EESSI/docs#26]
[https://eessi.github.io/](https://eessi.github.io/) (without `docs/`) now returns a 404 error page. We probably should put something (anything) there, or redirect to [https://eessi.github.io/docs/](https://eessi.github.io/docs/) or [https://www.eessi-hpc.org/](https://www.eessi-hpc.org/)?

**Comment by boegel** [comment]
We could fix this by renaming this `docs` repository to `eessi.github.io` (which is how you set up a website on the landing page).

If this breaks https://eessi.github.io/docs` (I'm hoping it will result in an automatic redirect, but I'm not sure), then we can fix that by adding a new `docs` repository that basically just forwards to `https://eessi.github.io`?

I'm not sure if that's the best way forward...

**Comment by boegel** [comment]
Or we set up a trivial landing page at `https://eessi.github.org` by making a new repo named `eessi.github.io` that just funnels to https://eessi.github.io/docs (or provides contact info like we already have there).

**Comment by boegel** [comment]
See https://pages.github.com

**Comment by robberteggermont** [comment]
I see it's also possible to create custom 404 pages, that could be another way to do it (and at the same time provide a fallback for typo's, renamed, etc.)?

**Comment by boegel** [comment]
Yeah, I like that idea...

Maybe a 404 page with a modified logo with a broken flask? :D

**Comment by robberteggermont** [comment]
I see we already have a somewhat working 404 for the docs/ subfolder
To get one for the root, we'll probably need to make a eessi.github.io repo?

**Comment by boegel** [comment]
Yes, that's probably the best option. I'll create it and give the docs team access to it.

**Comment by boegel** [comment]
@robberteggermont Repo created at https://github.com/EESSI/eessi.github.io + simplest possible landing page in place now, see https://eessi.github.io/
